deploy/dapr-multi-app/dapr.yaml

```diff
@@ -30,4 +30,20 @@ apps:
   - appID: pubsub-workflow
     appDirPath: ../../pubsub-workflow
     appPort: 3005
-    command: ["dotnet","run", "--DaprHTTPAppPort=3005"]
\ No newline at end of file
+    command: ["dotnet","run", "--DaprHTTPAppPort=3005"]
+  - appID: scheduler-jobs
+    appDirPath: ../../scheduler-jobs
+    appPort: 3006
+    command: [ "go","run", "scheduler-jobs.go"]
+  - appID: scheduler-actor-reminders-server
+    appDirPath: ../../scheduler-actor-reminders/server
+    appPort: 3007
+    command: [ "go","run", "player-actor.go"]
+  - appID: scheduler-actor-reminders-client
+    appDirPath: ../../scheduler-actor-reminders/client
+    appPort: 3008
+    command: [ "go","run", "player-actor-client.go"]
+  - appID: scheduler-workflow
+    appDirPath: ../../scheduler-workflow
+    appPort: 3009
+    command: [ "go","run", "scheduler-workflow.go"]
```

Comment:
```suggestion
    command: [ "go","run", "scheduler-workflow.go", "--DaprHTTPAppPort=3009"]
```

Reply:
please see my response explaining it for the scheduler-jobs example.